Assets/Scripts/Actors/ActorComponents/ActorPhysics.cs

```diff
@@ -380,77 +395,92 @@ public void JumpCheck()
 				jumpColCheckTimer = 0.0f;
 				//actor.GetAnimator().SetBool("isSliding", false);
 			}
-			else if ( jumpColCheckTimer > jumpColCheckTime && isOnGround && !IsInState( ActorStates.Rolling ) )
-			{
-				if ( !IsInState( ActorStates.Grounded ) )
-				{
-					// if !launching
-					ChangeState( ActorStates.Grounded );
-					stopMoveTimer = 0f;
-					//GainControl();
-				}
-
-				actor.animator.SetBool( "isJumping", false );
```

Comment:
I'm not totally sure what's going on here but it looks like it's a case where it is exiting the jump state and reentering grounded. So it's not triggering the jump animation, it's stopping it. Does it still seem necessary?